tests/server.js

```diff
@@ -1,142 +1,196 @@
-'use strict'
+"use strict";
```

Comment:
Another nit that I know will be resolved, but make sure any stylistic preferences are fixed in future commits so that the history stays mostly clean and just what was changed.